examples/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
+
+# cuvs empty project template build script
+
+# Abort script on first error
+set -e
+
+PARALLEL_LEVEL=${PARALLEL_LEVEL:=`nproc`}
+
+BUILD_TYPE=Release
+BUILD_DIR=build/
+
+CUVS_REPO_REL=""
+EXTRA_CMAKE_ARGS=""
+set -e
+
+
+if [[ ${CUVS_REPO_REL} != "" ]]; then
+  CUVS_REPO_PATH="`readlink -f \"${CUVS_REPO_REL}\"`"
+  EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DCPM_cuvs_SOURCE=${CUVS_REPO_PATH}"
```

Comment:
We might want to provide either a default value for `CUVS_REPO_PATH`or document it in the readme of the example. I didn't check, but are we documenting the build.sh example in the example readme?

Reply:
I added a default where if a repo path is not provided then the examples will try to look for the build directory and pick up the shared libraries from there.